### ISSUE TITLE
Load pretrained model

### ISSUE THREAD

**Issue by Kevin2** [radionets-project/radionets#65]
After loading a pretrained model the pretrained losses and learning rates are not added in the loss/lr plots.

**Comment by FritzGeise** [comment]
Referred in #77. Losses are plotted. Not sure about the learning rates tough.

**Comment by FritzGeise** [comment]
Update: Now the learning rates are also saved and added to loss/lr plots while using pre-trained models in #77 .
